scripts/make/check_changelogs.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import io
+import hashlib
+import os
+import requests
+
+SUPPORTED_VERSIONS = ["6.8", "7.8", "7.x"]
```

Comment:
looks like our version is not supported so this script will do nothing?

Reply:
removed